lib/classes/PluginManager.js

```diff
@@ -309,6 +310,41 @@ class PluginManager {
     });
   }
 
+  loadVariableResolvers(pluginInstance) {
+    const pluginName = pluginInstance.constructor.name;
+    for (const [variablePrefix, resolverOrOptions] of _.entries(
+      pluginInstance.variableResolvers || {}
+    )) {
+      let options = resolverOrOptions;
+      if (_.isFunction(resolverOrOptions)) {
+        options = {
+          resolver: resolverOrOptions,
+        };
+      }
```

Comment:
It'll be also good to validate that `resolverOrOptions` is an object if it's not a function